lib/sensu/api/process.rb

```diff
@@ -794,6 +794,48 @@ def resolve_event(event_json)
           end
         end
       end
+
+      aget %r{^/results?/\*/([\w\.-]+)/?$} do |check_name|
+        keys = "result:*:#{check_name}"
+        settings.redis.keys(keys) do |keys|
```

Comment:
That is correct. The KEYS command should only be used for debugging purposes. More information (including potential workarounds) at http://redis.io/commands/KEYS.